### PR TITLE
fix: [CDS-40428]: fix active service instance sync pop-up display

### PR DESCRIPTION
**Pull request by vikrant-harness**
### Summary

fix active service instance sync pop-up display

#### Screenshots

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by vikrant-harness** [comment]
retrigger titlecheck

**Comment by vikrant-harness** [comment]
trigger cypress

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancePopover.tsx [hunk 1]
```diff
@@ -109,19 +110,39 @@ export const ActiveServiceInstancePopover: React.FC<ActiveServiceInstancePopover
...
-          value: instanceData.podName || ''
-        },
-        {
-          label: getString('cd.serviceDashboard.artifact'),
-          value: instanceData.artifactName || ''
-        }
-      ]
+      values:
+        instanceData.instanceInfoDTO?.type === ServiceDeploymentType.AzureWebApp
+          ? [
```
Comment (YounixIO) [inline src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancePopover.tsx:115]: Move to a function

Comment (vikrant-harness) [reply src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancePopover.tsx]: @YounixIO Moved in the recent commit

**Review by sonaligoyal-harness — APPROVED** [review]

**Review by YounixIO — APPROVED** [review]

**Pull request merged**